rshell/main.py

```diff
@@ -1732,6 +1732,9 @@ def __init__(self, filename=None, timing=False, **kwargs):
         self.quit_serial_reader = False
         readline.set_completer_delims(DELIMS)
 
+        # add a method to call the user's editor by its name
+        setattr(self, 'do_' + EDITOR, self.do_edit)
```

Comment:
The way you coded this, you're adding do_vi (or whatever) to self, which is an instance of Shell. Normally, class methods would be a member of the Shell class, rather than a member of the Shell instance.

If you change this to `setattr(self.__class__, ....` then you should see that the `vi` command shows up in the help.

Reply:
Thanks, it works! Impressed!

I wanted to change the `__doc__` but it seems forbidden:

`AttributeError: attribute '__doc__' of 'method' objects is not writable`

I wanted to replace the `edit` keyword in the doc by the EDITOR name. But maybe it's a detail. This patch is useful enough (at least for me, I was always using my editor name instead of `edit`)

Should I pull request again?